wordpress-mobile/WordPress-iOS
# [Stats Refresh] Stat Row: add percentage data bar

Fixes #10708 , #10707

- Primarily, this adds a data percentage bar to the `StatsTotalRow`, currently only used by Insights Comments. 
- It also fixes the issue where hidden UI elements became visible on rotation.

---
To test:

Data bar:
- On a site with Comments, go to Stats > Insights > Comments.
  - Tip: en.blog has lots.
- Both tabs, `Authors` and `Posts and Pages`, show the bar.
- The first row is always 100%.
- The remaining rows are a percent of the first row.

UI elements:
- On rotation, verify hidden elements don't appear. (i.e. cell header ellipsis, row disclosure icon, data bar.)

---
![iphone_authors](https://user-images.githubusercontent.com/1816888/50744191-17219700-11de-11e9-8973-715317a552f5.png)

---
![iphone_pandp](https://user-images.githubusercontent.com/1816888/50744195-1c7ee180-11de-11e9-9cfe-7512705b9fa7.png)

---
![ipad_authors](https://user-images.githubusercontent.com/1816888/50744230-797a9780-11de-11e9-8f2d-0e2cc3d613ff.png)

---
![ipad_pandp](https://user-images.githubusercontent.com/1816888/50744237-7f707880-11de-11e9-9667-dc2d6bce0579.png)
